### PR TITLE
Fix typos found with codespell

### PR DESCRIPTION
**Pull request by xdelaruelle**
Fix typos found with the codespell utility [1].

[1] https://github.com/codespell-project/codespell

### CODE REVIEW

File: doc/sphinx/config.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following table describes available ``clush`` config file settings.
 |                 | :ref:`run modes <clushmode-config>` that can then  |
 |                 | be activated with `--mode`. All other ``clush``    |
 |                 | config file settings defined in this table might   |
-|                 | be overriden in a run mode. Each mode section      |
+|                 | be overridden in a run mode. Each mode section      |
```
Comment (thiell) [inline doc/sphinx/config.rst:51]: unaligned table might break sphinx

File: tests/TaskLocalMixin.py [hunk 2]
```diff
@@ -568,7 +568,7 @@ def testTaskAbortSelf(self):
 
         # retry
         task = task_self()
-        worker = task.shell("/bin/echo shouldnt see that")
+        worker = task.shell("/bin/echo shouldn't see that")
```
Comment (thiell) [inline tests/TaskLocalMixin.py:571]: we should just avoid the simple quote here, to make things safer

Comment (xdelaruelle) [reply tests/TaskLocalMixin.py]: ok, I propose to change *shouldn't* into *should not* in revised commit.

**Comment by thiell** [comment]
Prefect, thanks @xdelaruelle!

**Pull request merged**